uitest/Extensions/WindowsDriverExtensions.cs

```diff
@@ -0,0 +1,139 @@
+﻿// Copyright (c) Microsoft Corporation and Contributors
+// Licensed under the MIT license.
+
+using OpenQA.Selenium;
+using OpenQA.Selenium.Appium.Windows;
+using OpenQA.Selenium.Support.UI;
+
+namespace DevHome.UITest.Extensions;
+public static class WindowsDriverExtensions
+{
+    private const int DefaultTimeoutInMS = 60_000;
+    private const int DefaultPollingIntervalInMS = 500;
+
+    /// <summary>
+    /// Create an instance of <see cref="DefaultWait{T}"/>
+    /// </summary>
+    /// <param name="driver">Driver instance</param>
+    /// <param name="timeout">Wait timeout</param>
+    /// <param name="pollingInterval">Action interval</param>
+    /// <returns>Instance of <see cref="DefaultWait{T}"/></returns>
+    public static DefaultWait<WindowsDriver<WindowsElement>> Wait(
+        this WindowsDriver<WindowsElement> driver,
+        TimeSpan timeout,
+        TimeSpan pollingInterval)
+    {
+        return new (driver)
+        {
+            Timeout = timeout,
+            PollingInterval = pollingInterval,
+        };
+    }
+
+    /// <summary>
+    /// Create an instance of <see cref="DefaultWait{T}"/>
+    /// </summary>
+    /// <param name="driver">Driver instance</param>
+    /// <param name="timeoutInMS">Wait timeout in milliseconds</param>
+    /// <param name="pollingIntervalInMS">Action interval in milliseconds</param>
+    /// <returns>Instance of <see cref="DefaultWait{T}"/></returns>
+    public static DefaultWait<WindowsDriver<WindowsElement>> Wait(
+        this WindowsDriver<WindowsElement> driver,
+        int timeoutInMS = DefaultTimeoutInMS,
+        int pollingIntervalInMS = DefaultPollingIntervalInMS)
+    {
+        return driver.Wait(TimeSpan.FromMilliseconds(timeoutInMS), TimeSpan.FromMilliseconds(pollingIntervalInMS));
+    }
+
+    /// <summary>
+    /// Create an instance of <see cref="DefaultWait{T}"/> which ignores all <see cref="WebDriverException"/> exceptions
```

Comment:
Why do we want this?

Reply:
The [DefaultWait class](https://github.com/SeleniumHQ/selenium/blob/b5435553740e0d9ee1e8d402d211b0c63651c96f/dotnet/src/webdriver/Support/DefaultWait%7BT%7D.cs#L202C25-L202C25) gives you the option to ignore multiple exceptions while waiting, and fail the test on unexpected exceptions. This is useful for example if we want to ignore the exception thrown if an element was not found (e.g using `driver.FindElement*()`), allowing the `Until` to continue retrying until it timeouts, otherwise it will just consider that exception as an unexpected failure.